### ISSUE TITLE
Training traffic sign recognition with faster rcnn using ga

### ISSUE THREAD

**Issue by Zhihao1002** [ahmedfgad/GeneticAlgorithmPython#280]
Hi, 

I would like to ask if I want to integrate the ga to optimize the faster rcnn model's accuracy, could you explain a little on how i can modify the code to integrate GA into the model.

**Comment by ahmedfgad** [comment]
If you mean training the model using GA, then I highly recommend using pygad.kerasga (for Keras models) or pygad.torchga (for PyTorch models). But it is important to check that all the layers are compatible with the libraries so that their weights can be set and get by PyGAD.
